hardware/include/gen_if.vh

```diff
@@ -0,0 +1,4 @@
+   //General Interface Signals (do not remove indentation)
```

Comment:
This file already exists in iob lib

Reply:
The reset signal doesn't have the same name (reset in iob-cache and rst in iob-lib)